### PR TITLE
Ensure we await the call to kill the daemon

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

Noticed an issue where the daemon was only **sometimes** getting killed. I believe this was due to us not awaiting the exec that we spawned. I've turned the spawn into execs that will run synchronously. Tested on my M1 Mac: 
2021-11-18 at 6 05 PM

## Relevant release note information

Release Notes:

## Related JIRA tickets

Relates to JIRA: CWC-1264

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [x] No

If yes, please explain:

### CODE REVIEW

**Comment by sidpremkumar** [comment]
@lipmas can you confirm everything is still working on linux?

File: src/services/kube/kube.service.ts [hunk 1]
```diff
@@ -65,9 +66,9 @@ export async function killDaemon(configService: ConfigService) {
     if ( kubeConfig['localPid'] != null) {
         // First try to kill the process
         if (process.platform === 'win32') {
-            spawn('taskkill', ['/F', '/T', '/PID', kubeConfig['localPid'].toString()]);
+            exec(`taskkill /F /T /PID ${kubeConfig['localPid'].toString()}`);
         } else if (process.platform === 'linux') {
-            spawn('pkill', ['-s', kubeConfig['localPid'].toString()]);
+            exec(`pkill -s' ${kubeConfig['localPid'].toString()}`);
```
Comment (lipmas) [inline src/services/kube/kube.service.ts:71]: Got an error when running on linux:

```
➜  zli git:(fix/await-zli-disconnect-exec) ./bin/zli-linux tunnel cwc-dev-developer@dev-cluster
Logged in as: sebby@commonwealthcrypto.com, bzero-id:e30303b2-9048-4bab-9c24-9f5528dae59f, session-id:6d5dfd5b-fedc-4b85-81bd-59fe7aaccec7
Started kube daemon at localhost:40733 for cwc-dev-developer@dev-cluster
➜  zli git:(fix/await-zli-disconnect-exec) lsof -i tcp:40733      
COMMAND   PID  USER   FD   TYPE  DEVICE SIZE/OFF NODE NAME
daemon  12849 sebby    8u  IPv6 2174722      0t0  TCP *:40733 (LISTEN)
➜  zli git:(fix/await-zli-disconnect-exec) lsof -i tcp:40733
COMMAND   PID  USER   FD   TYPE  DEVICE SIZE/OFF NODE NAME
daemon  12849 sebby    8u  IPv6 2174722      0t0  TCP *:40733 (LISTEN)
➜  zli git:(fix/await-zli-disconnect-exec) ./bin/zli-linux disconnect
Logged in as: sebby@commonwealthcrypto.com, bzero-id:e30303b2-9048-4bab-9c24-9f5528dae59f, session-id:6d5dfd5b-fedc-4b85-81bd-59fe7aaccec7
/bin/sh: 1: Syntax error: Unterminated quoted string
Command failed: pkill -s' 12848
/bin/sh: 1: Syntax error: Unterminated quoted string
```

Comment (lipmas) [reply src/services/kube/kube.service.ts:71]: I think there is an extra `'` after pkill -s

Comment (sidpremkumar) [reply src/services/kube/kube.service.ts:71]: Yup my fault. I'll fix tomorrow and re request a review

Comment (lipmas) [reply src/services/kube/kube.service.ts]: yup that was it i pushed a fix and retested and it works:

```
➜  zli git:(fix/await-zli-disconnect-exec) ✗ ./bin/zli-linux disconnect
Logged in as: sebby@commonwealthcrypto.com, bzero-id:e30303b2-9048-4bab-9c24-9f5528dae59f, session-id:6d5dfd5b-fedc-4b85-81bd-59fe7aaccec7
Killed local Kube daemon
➜  zli git:(fix/await-zli-disconnect-exec) ✗ lsof -i tcp:40733                                   
➜  zli git:(fix/await-zli-disconnect-exec) ✗ 
```

Comment (sidpremkumar) [reply src/services/kube/kube.service.ts]: Oh ur the best thank you!!!!

**Review by lipmas — APPROVED** [review]

**Pull request merged**